### DISCUSSION TITLE
Unable to install v5.20

### DISCUSSION THREAD

**Discussion by talkleasy** [vinodsr/node-red-contrib-tuya-smart-device#123] [General]
I cannot upgrade my installation of this node, with following error messgae:
2023-06-12T12:11:45.666Z Install : node-red-contrib-tuya-smart-device 5.2.0

2023-06-12T12:11:45.919Z npm install --no-audit --no-update-notifier --no-fund --save --save-prefix=~ --production --engine-strict node-red-contrib-tuya-smart-device@5.2.0
2023-06-12T12:11:52.391Z [out] 
2023-06-12T12:11:52.391Z [out] up to date in 5s
2023-06-12T12:11:52.413Z rc=0

**Comment by vinodsr** [comment]
Please make sure you have node red version 2.5 or above



On Mon, 12 Jun 2023 at 5:43 PM, Andrew Cox ***@***.***> wrote:

> I cannot upgrade my installation of this node, with following error
> messgae:
> 2023-06-12T12:11:45.666Z Install : node-red-contrib-tuya-smart-device 5.2.0
>
> 2023-06-12T12:11:45.919Z npm install --no-audit --no-update-notifier
> --no-fund --save --save-prefix=~ --production --engine-strict
> ***@***.***
> 2023-06-12T12:11:52.391Z [out]
> 2023-06-12T12:11:52.391Z [out] up to date in 5s
> 2023-06-12T12:11:52.413Z rc=0
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vinodsr/node-red-contrib-tuya-smart-device/discussions/123>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADQ6OHNM6KBRGVLVTYNU2LXK4BVZANCNFSM6AAAAAAZDJAUJQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <vinodsr/node-red-contrib-tuya-smart-device/repo-discussions/123@
> github.com>
>
-- 
Vinod Surendran R
vinodsr.dev

**Comment by talkleasy** [comment]
Resolved – thanks for the quick response.
